org.gnome.GScope.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "org.gnome.GScope",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "command": "gscope",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
```

Comment:
Those cflags & cxxflags are not needed anymore on GNOME Runtime 3.30

Reply:
ok